Sources/Paywalls/Components/Common/PaywallComponentPropertyTypes.swift

```diff
@@ -171,19 +171,69 @@ public extension PaywallComponent {
 
     }
 
-    enum WidthSizeType: String, Codable, Sendable, Hashable, Equatable {
-        case fit, fill, fixed
+    struct Size: Codable, Sendable, Hashable, Equatable {
+
+        public let width: SizeConstraint
+        public let height: SizeConstraint
+
+        public init(width: PaywallComponent.SizeConstraint, height: PaywallComponent.SizeConstraint) {
+            self.width = width
+            self.height = height
+        }
+
     }
 
-    struct WidthSize: Codable, Sendable, Hashable, Equatable {
+    enum SizeConstraint: Codable, Sendable, Hashable {
+
+        case fit
+        case fill
+        case fixed(Int)
```

Comment:
Nit: Maaaaybe consider making this a UInt to get rid of the possibility of negative values?